### ISSUE TITLE
How to enable Swiping ? and How to modify the styling of the text that's on top of the rating?

### ISSUE THREAD

**Issue by aziz-boudi4** [Monte9/react-native-ratings#33]
Hi, Thanks for this component

I have few questions:
1- Swiping to change the rating doesn't seem to work, and I don't see any prop related to swiping even thought swiping is mentioned on the Readme file

2- How to modify the styling of the text that appears on top of the Airbnb rating?

Thanks

**Comment by Monte9** [comment]
yes, swipping and tapping are two different component right now. 

It would be awesome to consolidate both of these into 1 component.

**Comment by aziz-boudi4** [comment]
@Monte9 I'll give it a try and If I end up with a good result, I'll let you know

**Comment by Monte9** [comment]
ok, great! Looking forward to it! 👍

**Comment by VirenMohindra** [comment]
@aziz-boudi4 any update? did you manage to combine them?
